src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../Icon';
+import { Popover } from '../Popover';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+
+/**
+ * FieldLevelHelp Component for Patternfly React
+ */
+const FieldLevelHelp = ({ children, contentType, content, ...props }) => {
+  const trigger = contentType === 'popover' ? 'click' : 'hover focus';
+  const htmlContent = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: content
+      }}
+    />
+  );
+  const overlay =
+    contentType === 'popover' ? (
+      <Popover id="{contentType}">{htmlContent}</Popover>
+    ) : (
+      <Tooltip id="{contentType}">{htmlContent}</Tooltip>
```

Comment:
:+1: to removing tooltip here, its not part of the pf-design.

Reply:
Not removing the tooltip option in this iteration.
Keeping it temporarily for demo purposes, to explain an issue related to Formgroups and Popover below - https://github.com/patternfly/patternfly-react/pull/179#discussion_r164255291